packages/@sfx/sayt-plugin/test/unit/common/index.test.ts

```diff
@@ -0,0 +1,20 @@
+import { expect, stub } from '../../utils';
```

Comment:
This was a source of great pain for me. I didn't manage to get the same file running in Karma and Mocha.

Reply:
Plenty of issues.... Every time we added a new tool for testing it would break.